src/org/zaproxy/zap/extension/ruleconfig/RuleConfigParam.java

```diff
@@ -53,6 +53,11 @@
      * when checking for issues.
      */
     public static final String RULE_COOKIE_IGNORE_LIST = "rules.cookie.ignorelist";
+    
+    /**
+     * The default time, in seconds, to use for time-based attacks
+     */
+    public static final int RULE_DEFAULT_COMMON_SLEEP_TIME = 15;

```

Comment:
Wroth adding a see JavaDoc tag to `RULE_COMMON_SLEEP_TIME` and the other way around.